app/templates/gulpfile.js

```diff
@@ -213,8 +212,7 @@ function startTestServer() {
   });
 
   watch('app/scripts/**/*.js', scripts);
-  watch(['test/spec/**/*.js', 'test/index.html']).on('change', server.reload);
-  watch('test/spec/**/*.js', lintTest);
+  watch(['test/spec/**/*.js', 'test/index.html'], lintTest);
```

Comment:
Why is this part changed? I suggest returning it to the way it was:

```suggestion
  watch(['test/spec/**/*.js', 'test/index.html']).on('change', server.reload);
  watch('test/spec/**/*.js', lintTest);
```

Reply:
see below.

Reply:
You're correct that `lintBase` already reloads the server, but it doesn't reload when modifying `test/index.html` so I recommend keeping that.

As for your second change, why would changing `test/index.html` need to trigger `lintTest`? Is that on purpose?

Reply:
This is my updated code suggestion:

```suggestion
  watch('test/index.html').on('change', server.reload);
  watch('test/spec/**/*.js'], lintTest);
```

Reply:
Alternatively, remove the `server.reload` part from `lintBase`, that doesn't make much sense anyway and it would probably make the reloading behavior more obvious. Your choice.

Reply:
splendid, let me suggest another change in line with this and see where it goes :) we're getting there :D

Reply:
Thanks! ❤️ I hope this is not getting tedious for you.

Reply:
ha ha no probs ;)